compiler/rustc_borrowck/src/diagnostics/region_name.rs

```diff
@@ -179,6 +179,25 @@ impl RegionName {
             RegionNameSource::Static => {}
         }
     }
+
+    pub(crate) fn highlight_region_name_suggestion(&self, diag: &mut Diagnostic) {
```

Comment:
The name `highlight_region_name_suggestion` should be named something a bit more descriptive. In this case, perhaps something like `suggest_move_on_borrowing_closure`, but lots of names work.

Reply:
Sure, `suggest_move_on_borrowing_closure` is a meaningful name for this, since it is used for adding suggestion for cases when a nested closure outlives borrowed variables.